docs/slack-apps/slack-packages.md

```diff
@@ -0,0 +1,65 @@
+---
+id: slack-packages
+title: Slack Packages
+sidebar_label: Slack Packages
+---
+
+We've built in a variety of functions to make it easier to build Dark apps with Slack. Use these functions from autocomplete (dark/slack/Slack::), the same way you would any other function.
+
+![Slack](assets/packages/slackpackage.png)
+
+### getAllConversations
+
+Takes your Slack token as a String and returns a list of Dicts containing both your public and private Slack channel names and ids.
+
+![Slack](assets/packages/getAllConversations.png)
+
+### getConversations
+
+Takes your Slack token as a String and returns a list of Dicts containing your public Slack channel names and ids.
+
+![Slack](assets/packages/getConversations.png)
+
+### getConversationsWithDetails
+
+Takes your Slack token as a string and returns a list of Dicts containing the channel name, id, created at date (in Epoch seconds), creator, count of members, purpose and current topic. Does not include archived channels.
+
+![Slack](assets/packages/getConversationsWithDetails.png)
+
+### getNewConversations
```

Comment:
I think we should convert the results of these back into Dark constructs. So for example, the created_at dates should be converted into `Dates`.